default/python/AI/ShipDesignAI.py

```diff
@@ -1018,7 +1032,7 @@ def _class_specific_filter(self, partname_dict):
         pass
 
     def optimize_design(self, additional_parts=(), additional_hulls=(),
-                        loc=None, verbose=False, consider_fleet_count=True):
+                        loc=None, verbose=False, consider_fleet_count=False):
```

Comment:
Yes, right. Probably this should not be in this PR. It's some time ago I changed this, can't exactly recall why I did it.